ksp_plugin/interface.cpp

```diff
@@ -705,6 +705,27 @@ Iterator* principia__RenderedVesselTrajectory(Plugin const* const plugin,
       plugin));
 }
 
+void principia__ReportCollision(Plugin const* const plugin,
+                                char const* const vessel1_guid,
+                                char const* const vessel2_guid) {
+  journal::Method<journal::ReportCollision> m({plugin,
+                                               vessel1_guid,
+                                               vessel2_guid});
+  CHECK_NOTNULL(plugin)->ReportCollision(vessel1_guid, vessel2_guid);
+  return m.Return();
+}
+
+char const* CDECL principia__PileUpInfo(Plugin const* const plugin) {
```

Comment:
"Info" is ugly as hell, it means nothing.  Should probably be "DebugString".  Also wondering if we need this function at all now that it works.